buildstockbatch/postprocessing.py

```diff
@@ -42,6 +42,8 @@ def read_data_point_out_json(fs, reporting_measures, filename):
     try:
         with fs.open(filename, "r") as f:
             d = json.load(f)
+        if not d:
+            return None
     except (FileNotFoundError, json.JSONDecodeError):
         return None
```

Comment:
I'm kind of curious what kind of file would run into this block. We're already handling the file being missing and it not being valid json, which is the error hit when the file is just empty. Based on messing around a little with it, I think the file would have to be just

```json
null
```

I wonder where we're creating files like that?

Reply:
I ran into a case where it was just "{}" when ResStock fails in some specific step.